data/mods/ebook_lua/items/tools.json

```
@@ -0,0 +1,9 @@
+[
+  {
+    "id": "eink_tablet_pc",
+    "type": "TOOL",
+    "name": { "str": "e-ink tablet PC" },
+    "copy-from": "eink_tablet_pc",
+    "use_action": [ "EINKTABLETPC", "TOGGLE_UPS_CHARGING", "LUA_EBOOK" ]
```

Should use `extends` so we don't overwrite potential future base game updates to the tablet

https://docs.cataclysmbn.org/en/mod/json/reference/items/json_inheritance/#extends

```suggestion
    "extend": { "use_action": [ "LUA_EBOOK" ] }
```

---

Strangely, `extend` doesn't work on `extend`. Therefore I overwrote it instead.